package.json

```diff
@@ -1,38 +1,38 @@
 {
-	"name": "batch-10-tripen",
-	"version": "1.0.0",
-	"description": "A MERN travel timeline application",
-	"main": "index.js",
-	"repository": "https://github.com/pesto-students/batch-10-tripen.git",
-	"license": "MIT",
-	"husky": {
-		"hooks": {
-			"pre-commit": "lint-staged",
-			"pre-push": "lint-staged"
-		}
-	},
-	"scripts": {
-		"dev:client": "yarn --cwd client run start",
-		"dev:server": "yarn --cwd server run start",
-		"dev": "concurrently \"yarn dev:client\" \"yarn dev:server\"",
-		"lint": "./node_modules/.bin/eslint '**/*{.js,.jsx}'"
```

Comment:
changed .node_modules/.bin -> eslint in this case both would produce the same output.